packages/cli-plugin-scaffold-full-stack-app/templates/essentials/code/webiny.config.ts

```diff
@@ -30,6 +30,7 @@ export default {
                 getStackOutput({
                     folder: "project-applications-path/api",
                     env: options.env,
+                    variant: options.variant,
```

Comment:
We should probably just delete this whole package. :D 
Maybe we can discuss this with Pavel.